GoogleForCreators/web-stories-wp
# Remove Branded Logos from Templates.

## Summary
Automates this a good portion of this process:
https://docs.google.com/document/d/1P4yRs38DnQQP0gS06-Y-IEJvTJ8kbaqVTbxrgwoyFCM/edit#

Andd removes branded logos from all templates.

To create a template simply copy the raw story data json into: 
```
assets/src/dashboard/templates/raw/<template>.json
```  

and add data to `src/daschboard/templates/loadTemplate:56 getTemplates(imageBaseUrl)` function in the form of:
```
[template_name]: <template_json>
```

## Relevant Technical Choices
We decided to move to interpreting and updating resource urls from the source json to be a runtime calculation which avoids an intensive linting step and makes the process as easy as possible to create a template.

The parsing should all be memoized to the given baseUrl, so the client should only be doing this calculation on load.

## To-do
May need to remove branded images and titles.

## User-facing changes
- Removes logos from templates listed above.

## Testing Instructions
In the dashboard, go to `explore templates` and see that the `fashion`, `beauty`, 'cooking', 'travel', & `fitness` templates do not have branded logos in them.

## Screenshots
<img width="716" alt="Screen Shot 2020-06-03 at 11 08 13 AM" src="https://user-images.githubusercontent.com/35983235/83666681-8c87c780-a58a-11ea-9303-bbd595a8ddd5.png">



---

Fixes #2156 
Fixes #2159
